deploy/install/02-components/01-manager.yaml

```diff
@@ -77,6 +83,11 @@ spec:
       - name: longhorn
         hostPath:
           path: /var/lib/longhorn/
+      - name: longhorn-grpc-tls
+        secret:
+          secretName: longhorn-grpc-tls
+          optional: true
```

Comment:
So users need to provide the key-pair and ca if they want TLS enabled between manager and engine. If not provided, it's disabled by default?

Reply:
The user needs to create the `longhorn-grpc-tls` secret before the longhorn-manager pod is started.
Since the secret is marked optional it's fine if it's not there which means tls is disabled.